README.md

```diff
@@ -70,6 +70,17 @@ python -m mart experiment=CIFAR10_RobustBench \
 	fit=false \
 	+trainer.limit_test_batches=1 \
 	+attack@model.modules.input_adv_test=classification_eps8_pgd10_step1
+
+# train on 1 GPU
+python -m mart experiment=ArmoryCarlaOverObjDet_TorchvisionFasterRCNN \
+	trainer=gpu \
+	fit=true
+
+# train on multiple GPUs using Distributed Data Parallel
```

Comment:
Could you update with the example at https://github.com/IntelLabs/MART/pull/24#issuecomment-1351889109